Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/util/StringUtil.java

```diff
@@ -45,18 +45,16 @@
      * @param tokens
      * @return String
      */
-    public static String extractStringFromTokens(String tokenString, String tokens) {
+    public static String extractStringFromTokens(final String tokenString, final String tokens) {
         String resultString = "";
-        if (tokens != null && !tokens.isEmpty()) {
-            StringTokenizer tk = new StringTokenizer(tokenString, tokens);
-            StringBuffer outString = new StringBuffer();
-            while (tk.hasMoreTokens()) {
-                outString.append(tk.nextToken());
-            }
-            resultString = outString.toString();
-        } else {
-            resultString = tokenString;
+
+        final StringTokenizer tk = new StringTokenizer(tokenString, tokens);
+        final StringBuffer outString = new StringBuffer();
+        while (tk.hasMoreTokens()) {
```

Comment:
All existing locations where extractStringFromTokens(String,String) is called are passing non empty String values as parameters.  I can replace the condtion in the if statement to StringUtils.isNotEmpty(tokens) to check for an empty or null value.